pkg/resource/uri_test.go

```diff
@@ -30,56 +31,69 @@ var _ = Describe("uri", func() {
 			Expect(actual).To(Equal(expected))
 		},
 			Entry("Docker image", "https://gcr.io/scanning-customer/dockerimage@sha256:244fd47e07d1004f0aed9c156aa09083c82bf8944eceb67c946ff7430510a77b", &uriComponents{
-				name:    "https://gcr.io/scanning-customer/dockerimage",
-				version: "244fd47e07d1004f0aed9c156aa09083c82bf8944eceb67c946ff7430510a77b",
+				name:         "https://gcr.io/scanning-customer/dockerimage",
+				version:      "244fd47e07d1004f0aed9c156aa09083c82bf8944eceb67c946ff7430510a77b",
+				resourceType: pb.ResourceType_DOCKER,
 			}),
 			Entry("Debian package", "deb://lucid:i386:acl:2.2.49-2", &uriComponents{
-				name:    "acl",
-				version: "2.2.49-2",
+				name:         "acl",
+				version:      "2.2.49-2",
+				resourceType: pb.ResourceType_DEBIAN,
 			}),
 			Entry("Debian package without a specified distribution", "deb://arm64:build-essential:12.9", &uriComponents{
-				name:    "build-essential",
-				version: "12.9",
+				name:         "build-essential",
+				version:      "12.9",
+				resourceType: pb.ResourceType_DEBIAN,
 			}),
 			Entry("Generic file", "file://sha256:244fd47e07d1004f0aed9c156aa09083c82bf8944eceb67c946ff7430510a77b:foo.jar", &uriComponents{
-				name:    "foo.jar",
-				version: "244fd47e07d1004f0aed9c156aa09083c82bf8944eceb67c946ff7430510a77b",
+				name:         "foo.jar",
+				version:      "244fd47e07d1004f0aed9c156aa09083c82bf8944eceb67c946ff7430510a77b",
+				resourceType: pb.ResourceType_FILE,
 			}),
 			Entry("Maven package", "gav://ant:ant:1.6.5", &uriComponents{
-				name:    "ant:ant",
-				version: "1.6.5",
+				name:         "ant:ant",
+				version:      "1.6.5",
+				resourceType: pb.ResourceType_MAVEN,
 			}),
 			Entry("npm package", "npm://mocha:2.4.5", &uriComponents{
-				name:    "mocha",
-				version: "2.4.5",
+				name:         "mocha",
+				version:      "2.4.5",
+				resourceType: pb.ResourceType_NPM,
 			}),
 			Entry("scoped npm package", "npm://@babel/core:7.13.14", &uriComponents{
-				name:    "@babel/core",
-				version: "7.13.14",
+				name:         "@babel/core",
+				version:      "7.13.14",
+				resourceType: pb.ResourceType_NPM,
 			}),
 			Entry("NuGet", "nuget://log4net:9.0.1", &uriComponents{
-				name:    "log4net",
-				version: "9.0.1",
+				name:         "log4net",
+				version:      "9.0.1",
+				resourceType: pb.ResourceType_NUGET,
 			}),
 			Entry("pip package", "pip://raven:5.13.0", &uriComponents{
-				name:    "raven",
-				version: "5.13.0",
+				name:         "raven",
+				version:      "5.13.0",
+				resourceType: pb.ResourceType_PIP,
 			}),
 			Entry("RPM package", "rpm://el6:i386:ImageMagick:6.7.2.7-4", &uriComponents{
-				name:    "ImageMagick",
-				version: "6.7.2.7-4",
+				name:         "ImageMagick",
+				version:      "6.7.2.7-4",
+				resourceType: pb.ResourceType_RPM,
 			}),
 			Entry("RPM without a specified distribution", "rpm://el6:i386:ImageMagick:6.7.2.7-4", &uriComponents{
```

Comment:
This is in `main`, but I just realized this is a copypasta of the test case above. I think it's supposed to be this:

```suggestion
			Entry("RPM without a specified distribution", "rpm://i386:ImageMagick:6.7.2.7-4", &uriComponents{
```